### PR TITLE
bpo-44964: Add a note explaining the new semantics of f_last_i in frame objects

### PR DESCRIPTION
**Pull request by pablogsal**
https://bugs.python.org/issue44964

### CODE REVIEW

File: Doc/reference/datamodel.rst [hunk 1]
```diff
@@ -1069,8 +1069,9 @@ Internal types
       :attr:`f_code` is the code object being executed in this frame; :attr:`f_locals`
       is the dictionary used to look up local variables; :attr:`f_globals` is used for
       global variables; :attr:`f_builtins` is used for built-in (intrinsic) names;
-      :attr:`f_lasti` gives the precise instruction (this is an index into the
-      bytecode string of the code object).
+      :attr:`f_lasti` gives the precise instruction (it represents a wordcode index, which
```
Comment (markshannon) [inline Doc/reference/datamodel.rst:1072]: This is incorrect. f_lasti is the index in the bytecode. No need to multiply by two.

File: Doc/whatsnew/3.10.rst [hunk 2]
```diff
@@ -1948,6 +1948,12 @@ Changes in the C API
        source_buf = PyBytes_AsString(source_bytes_object);
        code = Py_CompileString(source_buf, filename, Py_file_input);
 
+  * For ``FrameObject`` objects, the ``f_lasti`` member now represents a wordcode
```
Comment (markshannon) [inline Doc/whatsnew/3.10.rst:1951]: Can you add a note to use PyFrame_GetLineNumber() instead

Comment (pablogsal) [reply Doc/whatsnew/3.10.rst:1951]: Fixed in https://github.com/python/cpython/pull/28208

File: Doc/reference/datamodel.rst (hunk 1 above)
Comment (pablogsal) [reply Doc/reference/datamodel.rst:1072]: Fixed in https://github.com/python/cpython/pull/28208

**Pull request merged**